### PR TITLE
 Render summary from data-service API on the curator-service home page.

### PR DESCRIPTION
**Pull request by axmb**
Couple things:

- Not in love with the solution to type HTTP client responses (e.g. duplicating the interface definition). It's probably worth having a published type for clients to ingest that API data (at least for Node clients), but I'm satisfied with this as a starting point.
- I'll catch up with Allyson on front end plans tomorrow. For now, just getting this on the home page as a placeholder.
- Vagrant set up would be great. ;-) To test locally, I manually ran a local MongoDB and both Node apps. Not terrible, but could be better.

### CODE REVIEW

File: verification/curator-service/src/controllers/home.ts [hunk 1]
```diff
@@ -1,10 +1,37 @@
 import { Request, Response } from 'express';
+import axios from 'axios';
+
+interface Case {
+    _id: string;
+    outcome: string;
+    date: Date;
+}
+
+function getLatestCaseDate(cases: [Case]): Date {
```
Comment (attwad) [inline verification/curator-service/src/controllers/home.ts:10]: I think you can use Cases.find() and provide a {'sort': {'date': -1} param to do what you want

File: verification/curator-service/src/controllers/home.ts [hunk 2]
```diff
@@ -1,10 +1,37 @@
 import { Request, Response } from 'express';
+import axios from 'axios';
+
+interface Case {
```
Comment (attwad) [inline verification/curator-service/src/controllers/home.ts:4]: yeah that duplication is certainly not great once those types will start to grow and we'll have to keep them in sync :( cases are already defined in model/case.ts so let's find a way to export what we need from there. (in a later PR, this is fine for now)

**Comment by attwad** [comment]
Also I started looking at docker compose for the dev stack, I have it running on my machine fine but I need to figure out the github action stuff, expect a PR later today (https://github.com/open-covid-data/healthmap-gdo-temp/pull/32)

File: verification/curator-service/src/controllers/home.ts (hunk 1 above)
Comment (axmb) [reply verification/curator-service/src/controllers/home.ts:10]: As with all problems in JS, `reduce` is the simplest I can make this.

File: verification/curator-service/src/controllers/home.ts (hunk 2 above)
Comment (axmb) [reply verification/curator-service/src/controllers/home.ts:4]: +1 -- we should discuss more

**Comment by axmb** [comment]
[Implement the curation case data routes and wire it to the data server](https://trello.com/c/xrkBAgQR/49-implement-the-curation-case-data-routes-and-wire-it-to-the-data-server)

**Pull request merged**